articles/what-is-jenkins/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/what-is-jenkins/
+title: What is Jenkins
+description: Developers use Jenkins to continuously build and test software projects, making it easier to integrate changes to the project and making it easier for users to obtain a good build with no errors and bugs.
+author: judy-nduati
+date: 2020-09-15T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/what-is-jenkins/hero.jpg
+    alt: Jenkins Continuous Integration tool
+---
+Jenkins is a Continuous Integration (CI) tool and an open-source tool written in Java. Jenkins achieves Continuous Integration with the help of plugins, which are over a thousand in count. Developers mostly use Jenkins for Continuous Integration because of its flexibility and the number of plugins it supports.
+<!--more-->
+### What is Jenkins?
+Developers use Jenkins to continuously build and test software projects, making it easier to integrate changes to the project, and making it easier for users to obtain a good build with no errors and bugs. Jenkins allows developers to continuously deliver a software project by integrating with a large number of development, deployment, and testing technologies such as Git, Maven, Puppet, Selenium, Ansible, and Nagios.
```

Comment:
Elaborate on builds and how they fit into the developer process. It might be beneficial to separate out the technologies by function- the way it is phrased right now makes it seem that Git, Maven, Puppet, etc. are all testing technologies.